netops_demo/py/functions/generate/generate.py

```diff
@@ -94,7 +94,7 @@ def _sites(context):
     for company in deployment.companies:
         company_name_label = _company_name_to_label(company.name)
 
-        for i, site_locations in enumerate(company.locations):
+        for i, site_locations in enumerate(company.locations.values()):
```

Comment:
What is this change? Does it fix something?

Reply:
Yes,
Instead of taking values every time from the specific device's generator, values were taken for all the devices from a single generator.